python/doc/examples/functional_modeling/field_functions/plot_process_sample.py

```diff
@@ -149,7 +149,6 @@
     graph.add(curve)
 
 curve = ot.Curve(vertices, process_sample_mean, "process sample mean")
-curve.setColor("red")
```

Comment:
in this particular case, there are already n_samples curves on the graph, so removing this line will make the curve use an arbitrary color

Reply:
yes! But I saw that all the process samples are idential! It come sfrom the fact that the X_distribution is a JointDistribution composed of marginals which are Dirac distributions...:
```
>>>print(X_distribution)
JointDistribution(Dirac(point = [7.5e+08]), Dirac(point = [2.75e+09]), Dirac(point = [10]), IndependentCopula(dimension = 3))
```
 These marginals come from the Caboche model.

Reply:
yes! But I saw that all the process samples are idential! It come sfrom the fact that the X_distribution is a JointDistribution composed of marginals which are Dirac distributions...:
```
>>>print(X_distribution)
JointDistribution(Dirac(point = [7.5e+08]), Dirac(point = [2.75e+09]), Dirac(point = [10]), IndependentCopula(dimension = 3))
```
 These marginals come from the Caboche model.

Reply:
This example is not mentioned in the section 'Examples based on this usecase' in the Chaboche usecase! Actually, I did not manage to find it in the OT documentation....

Reply:
I don't know if you did anything, but it is now mentioned in the "Examples based on this usecase" section: see https://output.circle-artifacts.com/output/job/ad1e2baf-1c02-4e4a-9e1f-4703618ecc26/artifacts/0/html/usecases/use_case_chaboche.html#examples-based-on-this-use-case

Reply:
ok! It appears now!